Makefile.am

```diff
@@ -1,16 +1,19 @@
 
 ACLOCAL_AMFLAGS = -I m4
 
-#@INC_AMINCLUDE@ is recommented for automake but does not work correcly.
-#So comment it out for now.
-#@INC_AMINCLUDE@
+#@INC_AMINCLUDE@ is recommented for automake and doxygen
+#but there is an naming comflict. So we rename aminclude.am to
+#doxygen.am from srcdir with allows INC_AMINCLUDE to generate 
+#aminclude.am to blddir.
+@INC_AMINCLUDE@
+include $(top_srcdir)/doxygen.am
 
```

Comment:
The `@INC_AMINCLUDE` directive will cause configure to generate `aminclude.am` in the build dir.  That in itself is fine.  The only problem is that it causes `make dist` to fail when it tries to clean the working dir with `make dist clean`.

Adding the following line in this spot (and regenerating Makefile.in) will fix it:

```
distclean-local:
        rm $(top_builddir)/aminclude.am
```
(the eight spaces above need to be replaced with a tab, as required by make)

Reply:
Ok, the patch addresses this and implements the Fedora preference for separating licenses from doc.

Reply:
Thanks.